hazelcast/src/test/java/com/hazelcast/core/server/StartServerTest.java

```diff
@@ -28,7 +30,7 @@ public void testConstructor() {
     }
 
     @Test
-    public void testMain() {
+    public void testMain() throws FileNotFoundException {
```

Comment:
because start server tries to write port number to a file. It may throw exception.